sonar-db/src/test/java/org/sonar/db/component/ComponentDbTester.java

```diff
@@ -83,8 +83,13 @@ public void insertComponents(ComponentDto... components) {
     db.commit();
   }
 
-  public void indexProjects() {
+  public void indexProjectsAndViews() {
     dbClient.componentIndexDao().indexProjects();
```

Comment:
This method is only used for test = keeping it is a pain as it requires to maintain it.